### ISSUE TITLE
Update stable version in repo from 0.0.2 to 0.1.2

### ISSUE THREAD

**Issue by GermanBluefox** [TA2k/ioBroker.imow#5]
# Think about update stable version to 0.1.2
**Version**: stable=**0.0.2** (592 days old) => latest=**0.1.2** (15 days old)
**Installs**: stable=**72** (72.73%), latest=**14** (14.14%), total=**99**

Click to use [developer portal](https://www.iobroker.dev/adapter/imow/releases)
Click to [edit](https://github.com/ioBroker/ioBroker.repositories/edit/master/sources-dist-stable.json#L1163)

**Do not close this issue manually as a new issue will be created if condition for update still exists.**

Please drop a comment if any reason exists which blocks updating to version 0.1.2 at this time.

Note: This is an automatically generated message and not personally authored by bluefox!
      @mcm1957 for evidence

**Comment by mcm1957** [comment]
blocked by  https://github.com/TA2k/ioBroker.imow/issues/6
